Week-4/assignment4.2/teacher.js

```diff
@@ -0,0 +1,29 @@
+var Person = function() {};
+Person.prototype.initialize=function(name,age){
+    this.name=name;this.age=age;
+    };
+// TODO: create the class Teacher and a method teach
+//Document.setPrototypeOf(Teacher,Person);
+//"use strict";
+class Teacher extends Person{
```

Comment:
we should be using prototype inheritence and not class based inheritence

Reply:
inheritance achieved using prototype inheritance.